deployment/deployutils/deployutils.cpp

```diff
@@ -2031,16 +2046,25 @@ bool generateHardwareHeaders(const IPropertyTree* pEnv, StringBuffer& sbDefn, bo
     addItem(jsStrBuf, pEnv, XML_TAG_NAS,       TAG_DIRECTORY,       "", 0, 1, "", 1);
     addItem(jsStrBuf, pEnv, XML_TAG_NAS,       TAG_MASK,       "", 0, 1, "", 1);
     addItem(jsStrBuf, pEnv, XML_TAG_NAS,       TAG_TRACE,       "", 0, 1, "", 1);
+
+    addItem(jsStrBuf, pEnv, "cost",  "moneyLocale", "", 0, 0, "", 1);
+    addItem(jsStrBuf, pEnv, "cost",  "perCpu", "", 0, 0, "", 1);
+    addItem(jsStrBuf, pEnv, "cost",  "storageAtRest", "", 0, 0, "", 1);
+    addItem(jsStrBuf, pEnv, "cost",  "storageReads", "", 0, 0, "", 1);
+    addItem(jsStrBuf, pEnv, "cost",  "storageWrites", "", 0, 0, "", 1);
+
     jsStrBuf.append("compTabs['Hardware'][compTabs['Hardware'].length]= 'Computer Types';");
     jsStrBuf.append("compTabs['Hardware'][compTabs['Hardware'].length]= 'Switches';");
     jsStrBuf.append("compTabs['Hardware'][compTabs['Hardware'].length]= 'Domains';");
     jsStrBuf.append("compTabs['Hardware'][compTabs['Hardware'].length]= 'Computers';");
     jsStrBuf.append("compTabs['Hardware'][compTabs['Hardware'].length]= 'NAS';");
+    jsStrBuf.append("compTabs['Hardware'][compTabs['Hardware'].length]= 'cost';");
     jsStrBuf.append("compTabToNode['Computer Types']= 'ComputerType';");
     jsStrBuf.append("compTabToNode['Switches']= 'Switch';");
     jsStrBuf.append("compTabToNode['Domains']= 'Domain';");
     jsStrBuf.append("compTabToNode['Computers']= 'Computer';");
     jsStrBuf.append("compTabToNode['NAS']= 'NAS';");
+    jsStrBuf.append("compTabToNode['cost']= 'cost';");
```

Comment:
Cost might need to be capitalized based on local trends

Reply:
Follows yaml conventions.